app/jobs/create_default_admin_set_job.rb

```diff
@@ -0,0 +1,6 @@
+class CreateDefaultAdminSetJob < ActiveJob::Base
+  def perform(account)
+    AccountElevator.switch!(account.tenant)
```

Comment:
I'm not sure I understand enough to answer that.  That's is a question for @mjgiarlo, who wrote the original code in his PR #884.